project_task_schedule/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Sunflower IT <https://sunflowerweb.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Task Schedule",
+    "version": "13.0.1.0.1",
+    "category": "Project",
+    "author": "Sunflower IT, " "Odoo Community Association (OCA)",
```

Comment:
This has some weird apostrophe flavour, did you run `pre-commit run -a`? [pre-commit](https://odoo-community.org/blog/the-oca-blog-1/post/black-isort-pre-commit-97)